stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -251,6 +260,20 @@ public void onChildRemoved(Object parentElement, Object childElement) {
 
       removeElementTree(childElement);
     }
+
+    @Override
+    public void onInspectRequested(Object element) {
```

Comment:
Tomato, tomato.  I was thinking the direction of the verb was backwards in this case since it implied that the inspect request came from Stetho, rather than asynchronously answering an inspect request from Chrome.